SPECS/glide/glide.spec

```diff
@@ -0,0 +1,87 @@
+Summary:        Vendor Package Management for Goland
+Name:           glide
+Version:        0.13.3
+Release:        6%{?dist}
+License:        MIT
+URL:            https://github.com/Masterminds/glide
+# Source0:      https://github.com/Masterminds/%{name}/archive/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+Group:          Development/Tools
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+BuildRequires:  git
+BuildRequires:  golang
+BuildRequires:  perl
+BuildRequires:  ca-certificates
+
+%description
+Glide is a tool for managing the vendor directory within a Go package.
+
+%define OUR_GOPATH %{_topdir}/.gopath
+
+%prep
+%autosetup -p1
+
+%build
+export GO111MODULE=auto
+export GOPATH=%{OUR_GOPATH}
+mkdir -p ${GOPATH}/src/github.com/Masterminds/glide
+cp -r * ${GOPATH}/src/github.com/Masterminds/glide/.
+pushd ${GOPATH}/src/github.com/Masterminds/glide
+make VERSION=%{version} build %{?_smp_mflags}
+popd
+
+%check
+export GO111MODULE=auto
+export GOPATH=%{OUR_GOPATH}
+pushd ${GOPATH}/src/github.com/Masterminds/glide
+make test %{?_smp_mflags}
+popd
+
+%install
+export GO111MODULE=auto
+export GOPATH=%{OUR_GOPATH}
+pushd ${GOPATH}/src/github.com/Masterminds/glide
+make install %{?_smp_mflags}
+install -vdm 755 %{buildroot}%{_bindir}
+install -vpm 0755 -t %{buildroot}%{_bindir}/ ./glide
+popd
+
+%files
+%defattr(-,root,root)
+%{_bindir}/glide
+
+%changelog
+*   Fri Sep 17 2021 Suresh Babu Chalamalasetty <schalam@microsoft.com> - 0.13.3-6
+-   Initial CBL-Mariner import from Photon
```

Comment:
Please add after Photon: (license: Apache2) 
That was the license that we pulled the SPEC file under.

Reply:
Added.